test/test-lib/github-mock.ts

```diff
@@ -0,0 +1,52 @@
+import * as Octokit from '@octokit/rest';
+import * as Promise from 'bluebird';
```

Comment:
Does `Octokit` return a Bluebird enhanced promises?
If not then we should better remove the Bluebird import, to have more accurate test cases about the Promise types used.

Reply:
That's why I wrap the response in a Promise.try to have Bluebird promises to work with.